File: pys/ceoscoredaily.py
import requests
from bs4 import BeautifulSoup

url = 'http://www.ceoscoredaily.com/news/article_list_all.html'

req = requests.get(url)
bs = BeautifulSoup(req.text, 'lxml')

wrapper = bs.find('div', class_='article_list_002').find('ul')
items = wrapper.find_all('li')

result = list()
for item in items:
    temp = {
        'title': '',
        'link': '',
        'date': '',
        'content': ''
    }
    p_tag = item.find('p', class_='tit')
    a_tag = p_tag.find('a')

    temp['title'] = a_tag.get_text().strip()
    temp['link'] = 'http://www.ceoscoredaily.com/news/' + a_tag['href']
    temp['date'] = item.find('p', class_='date').get_text().strip()

    # contents
    req2 = requests.get(temp['link'])
    bs2 = BeautifulSoup(req2.text, 'lxml')
    wrapper2 = bs2.find('div', class_='article_body')
    #temp['contents'] = ' '.join([it.get_text().strip() for it in wrapper2.find_all('span', recursive=True)])
    temp['contents'] = wrapper2.get_text().strip().replace('\t', '').replace('\r', '').replace('\n', ' ')

    result.append(temp)

print(result)

File: crawlers/ceoscoredaily.go
package crawlers

import (
	"fmt"
	"net/http"

	"flashnews/models"
	"flashnews/utils"

	"github.com/PuerkitoBio/goquery"
)

const ceoScoreDailyCommonURL = "http://www.ceoscoredaily.com/news/article_list_all.html"
const ceoScoreDailyItemURL = "http://www.ceoscoredaily.com/news/"

type CEOScoreDaily struct{}

func (c CEOScoreDaily) GetName() string {
	return "ceoscoredaily"
}

func (c CEOScoreDaily) GetGroup() string {
	return "7"
}

func (c CEOScoreDaily) GetList(number int) ([]models.NewsItem, error) {
	// Number
	var _number int
	if number > 10 || number < 1 {
		_number = 10
	} else {
		_number = number
	}
	result := make([]models.NewsItem, _number)

	// Request
	req, err := http.Get(ceoScoreDailyCommonURL)
	if err != nil {
		return result, err
	}
	defer req.Body.Close()

	if req.StatusCode != 200 {
		return result, fmt.Errorf("[ERROR] Request Status Code : %d, %s", req.StatusCode, req.Status)
	}

	// HTML Load
	html, err := goquery.NewDocumentFromReader(req.Body)
	if err != nil {
		return result, err
	}

	// Parsing
	wrapper := html.Find("div.article_list_002").Find("ul")
	items := wrapper.Find("li")
	items.Each(func(i int, sel *goquery.Selection) {
		if i >= _number {
			return
		}

		pTag := sel.Find("p.tit")
		aTag := pTag.Find("a")
		href, ok := aTag.Attr("href")
		if !ok {
			result[i] = models.NewsItem{}
			return
		}
		url := ceoScoreDailyItemURL + href

		date := sel.Find("p.date").Text()
		title := aTag.Text()

		result[i] = models.NewsItem{
			Title:    title,
			URL:      url,
			Contents: "",
			Datetime: date,
		}
	})

	return result, nil
}

func (c CEOScoreDaily) GetContents(item *models.NewsItem) error {
	// Request
	req, err := http.Get(item.URL)
	if err != nil {
		return err
	}
	defer req.Body.Close()

	if req.StatusCode != 200 {
		return fmt.Errorf("[ERROR] Request Status Code : %d, %s", req.StatusCode, req.Status)
	}

	// HTML Load
	html, err := goquery.NewDocumentFromReader(req.Body)
	if err != nil {
		return err
	}

	// Parsing
	wrapper := html.Find("div.article_body")
	contents := utils.TrimAll(wrapper.Text())

	item.Contents = contents
	return nil
}
